InvenTree/report/api.py

```diff
@@ -16,11 +16,12 @@
 import InvenTree.helpers
 import order.models
 import part.models
-from InvenTree.mixins import ListAPI, RetrieveAPI, RetrieveUpdateDestroyAPI
+import report.models
+from InvenTree.mixins import (ListAPI, RetrieveAPI, RetrieveUpdateAPI,
+                              RetrieveUpdateDestroyAPI)
+from plugin.serializers import MetadataSerializer
 from stock.models import StockItem, StockItemAttachment
 
-from .models import (BillOfMaterialsReport, BuildReport, PurchaseOrderReport,
-                     SalesOrderReport, TestReport)
```

Comment:
Question: Why this rewrite?

Reply:
The reason is that I made a combined method for all Report metadata - where I  use getattr based on the argument.

`UseModel = getattr(report.models, f"{reporttype}Report")`

This is not possible (without some other, sketchy solutions using `sys`) if the modules are imported in the main namespace.

If you have a suggestion of how to change the `getattr` method, please let me know.

Reply:
I would be keener on a strongly typed reference. Names might change, this way refactoring will not work automatically. I will type a suggestion up.

Reply:
OK, please share if you have another solution! However, looking at the other imports let's the one I removed stand out as different:
```
import build.models
import common.models
import InvenTree.helpers
import order.models
import part.models
```

Reply:
> @miggland I have made a commit to [this branch](https://github.com/matmair/InvenTree/tree/miggland/metadata-hardreferences) that rolls back most of the import changes and uses hot refs for the view

Ah, good solution! I'll try to get back to this and incorporate your ideas in the next few days.